test/unit/test-amp-context.js

```diff
@@ -309,6 +313,81 @@ describes.sandboxed('3p ampcontext.js', {}, (env) => {
     return promise;
   });
 
+  it('Should return failure message on invalid resize requests made after invalid request before 500ms timeout', async () => {
+    win.name = generateSerializedAttributes();
+    const context = new AmpContext(win);
+
+    // Resetting since a message is sent on construction.
+    windowPostMessageSpy.resetHistory();
+
+    const successCallbackSpy = env.sandbox.spy();
+    const failureCallbackSpy = env.sandbox.spy();
+    const initialId = context.nextResizeRequestId_;
+
+    const height = 100;
+    const width = 200;
+
+    //initially, accept all requests
+    expect(context.blockRepeatedInvalidRequests_).equal(false);
+
+    const promise = Promise.all([
+      context
+        .requestResize(height, width)
+        .then(successCallbackSpy)
+        .then(() => {
+          expect(successCallbackSpy).to.be.calledOnce;
+        }),
+      context
+        .requestResize(height, width)
+        .catch(failureCallbackSpy)
+        .then(() => {
+          expect(failureCallbackSpy).to.be.calledOnce;
+        }),
+    ]).then(() => {
+      //will block requests for the next 500ms since the last request was invalid.
+      expect(context.blockRepeatedInvalidRequests_).equal(true);
```

Comment:
Instead of testing private properties can we do something like 
- freeze timer
- call resize request and expect success
- call resize request and expect to throw
- tick 500ms
- call resize request and expect sucess

Reply:
this should also get rid of the codecov warning